pages/1.10/release-notes/1.10.4/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.4
+title: Release Notes for 1.10.4
+menuWeight: 10
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.4.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.2/dcos_generate_config.sh?_ga=2.35647400.804658989.1513010878-1207888710.1504116514"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.10.4
+
+- CORE-1375 - Docker executor does not hang due to lost messages.
+- DCOS-1877 - DC/OS CA certificate bundle now gets propagated to public slaves.
```

Comment:
This is a wrong JIRA id `DCOS-1877` - Need to proper one to verify if the fix has gone in ( I couldn't verify that either).

Reply:
That was a typo; the correct JIRA ID is DCOS-18777. I've updated the RN accordingly.